### PR TITLE
Update backend docs

### PR DESCRIPTION
**Pull request by OmarIthawi**
### TODO

 - [x] Move the docs to this repo from the Appsembler openedx-docs repo.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
[(2) Update Trinity OAuth docs](https://trello.com/c/4Na2A5qr/2777-2-update-trinity-oauth-docs)

File: README.rst [hunk 1]
```diff
@@ -23,6 +23,10 @@ Please do not report security issues in public. Please email security@appsembler
 
 Getting Help
 ------------
+There are two docs about the Trinity project:
+
+ - [The Trinity Backend](https://github.com/appsembler/openedx-docs/blob/master/openedx/Trinity/OAuth2.0-integration.md)
```
Comment (johnbaldwin) [inline README.rst]: @OmarIthawi Please remove the indentation as this shows as a code block for these two bullets. Otherwise, LGTM!

Comment (johnbaldwin) [reply README.rst]: @OmarIthawi Also, these links go to a private repository, you might want to either move or copy them to this repo.

But we should really create an open developer docs repo for the docs we can share with the public (totally outside of the scope of this PR, of course)

**Review by johnbaldwin — CHANGES_REQUESTED** [review]
There are links pointing to a private repo

File: README.rst (hunk 1 above)
Comment (OmarIthawi) [reply README.rst]: I like the idea of moving the docs here! Thanks @johnbaldwin!

**Comment by OmarIthawi** [comment]
@johnbaldwin could you please take another look? I've update the documentation to require no links to private repositories.

**Comment by OmarIthawi** [comment]
@johnbaldwin this now should look a bit better.

**Comment by OmarIthawi** [comment]
Thanks @johnbaldwin for the first review, I've addressed all the comments, so probably safe to be merged.

**Pull request merged**